Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -36,16 +36,79 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
                 (await multipleAsync.ReadAsync<Entities.ContactPoliciesSettings>()).FirstOrDefault();
 
             if (contactPoliciesSettings == null)
+            {
                 return null;
+            }
+
             if (contactPoliciesSettings.IdUser == null)
+            {
                 return contactPoliciesSettings;
+            }
 
             contactPoliciesSettings.ExcludedSubscribersLists =
                 (await multipleAsync.ReadAsync<ExcludedSubscribersLists>()).ToList();
 
             return contactPoliciesSettings;
         }
 
+        public async Task<Entities.ContactPoliciesSettings> GetBasicContactPoliciesSettingsAsync(string accountName)
```

Comment:
Why do you add a new method? is `GetContactPoliciesSettingsAsync` not enough?

Reply:
Maybe we could lose some of these benefits and simplify the code. 

For example, 500 error is an option, but another option (by the moment) could be to silently do nothing (we will fix it when we add permissions validation.

Reply:
Let me check it better.

Reply:
Todo ese código me sigue haciendo ruido. Hablemoslo mañana, tiene que haber forma de simplificarlo o por lo menos hacerlo más autodocuemtable.

Marco esto como resuelto, se ve más fácil en https://github.com/FromDoppler/doppler-contact-policies-api/pull/21/files#r660898669